lib/tasks/route_import.rake

```diff
@@ -5,12 +5,74 @@ require 'csv'
 namespace :routes do
   task :import, [:csv_file] => :environment do |_, args|
     Route.delete_all
+    BusStopsRoute.delete_all
+    @route_hash = {}
```

Comment:
In general I'm curious why you used instance variables throughout.

Reply:
Idk

Reply:
Ok I'll do local scopes then